Analysis/Tasks/PWGDQ/tableReader.cxx

```diff
@@ -423,8 +423,33 @@ struct DQEventMixing {
           }
         } // end for (muon2)
       }   // end for (muon1)
-    }     // end for (event combinations)
-  }       // end process()
+
+      for (auto& track1 : tracks1) {
+        for (auto& muon2 : muons2) {
+          twoTrackFilter = track1.isBarrelSelected() & muon2.isMuonSelected();
```

Comment:
Here you have to be careful, since the electrons and muons are selected with different cuts, and possibly with a different number of cuts. At the moment at least, there is one single cut on the muons and possibly multiple ones on the electrons. So we must agree on a policy on how to combine these cuts. If we stay with just a single cut on the muon, then the bitwise AND should be changed here to a normal logical AND. Then, later when you fill electron-muon pair histograms, if you want to fill histograms separately for each electron cut, then you have to adapt the code. You should do it as in the electron-electron loop, where you must take into account the electron cuts fulfilled.